cc/imgproc/imgprocBindings.h

```diff
@@ -204,6 +204,95 @@ namespace ImgprocBindings {
 		  };
 	  };
   };
+
+	class Accumulate : public CvClassMethodBinding<Mat> {
+	public:
+		cv::Mat dst;
+		
+		void createBinding(std::shared_ptr<FF::Value<cv::Mat>> self) {
+			auto src = req<Mat::Converter>();
```

Comment:
I think dst should be the input parameter: `auto dst = req<Mat::Converter>();`, since src (the first Mat) is already passed via "self".

Reply:
I thought I needed to change the positions of dst and src for the method chain. also dst should be input(first parameter) and output.

So here the self parameter is dst and dst on line 210 is declared for return.

please see the last review's reply